app/src/main/java/com/kickstarter/viewmodels/projectpage/ProjectPageViewModel.kt

```diff
@@ -516,52 +534,56 @@ interface ProjectPageViewModel {
                     }
                 }.share()
 
-            val currentProject = Observable.merge(
+            val currentProject = Observable.mergeArray(
                 initialProject,
-                refreshedProjectNotification.compose(values()),
+                refreshedProjectNotification.compose(valuesV2()),
                 projectOnUserChangeSave,
-                savedProjectOnLoginSuccess,
-                projectOnDeepLinkChangeSave
+                projectOnDeepLinkChangeSave,
+                savedProjectOnLoginSuccess
             )
 
             var previousScreen = ""
-            intent()
+            intent
                 .subscribe { previousScreen = it.getStringExtra(IntentKey.PREVIOUS_SCREEN) ?: "" }
+                .addToDisposable(disposables)
 
             SendThirdPartyEventUseCase(sharedPreferences, ffClient)
-                .sendThirdPartyEvent(
+                .sendThirdPartyEventV2(
```

Comment:
You could have used `SendThirdPartyEventUseCaseV2().sendThirdPartyEvent`, instead of creating a new one, I'll suggest delete the new one created in this PR